pkg/provisioning/test/pipelines1/provision1-pipeline-with-processors.yml

```diff
@@ -1,29 +1,29 @@
-version: 1.0
+version: 2.0
```

Comment:
why did we change all the test files to version 2.0? aren't we still supporting version 1.0 too?

Reply:
To be honest, if I had the time I would go a step further, remove the test files and test this with a mocked parser that returns the configuration (we can do this now that we have the interface `config.Parser`). This would ensure we are testing the service code in isolation. After all, the service doesn't care about how the parser gets the configuration (from yaml, json, or if it makes it up 🙂), so it's not the job of these tests to ensure the parser works correctly. But I don't think it's worth it at this point.

We do support both versions, and that's still tested in the tests for `yaml.Parser`. It's just that for service tests we use the latest version now.